config.go

```diff
@@ -47,7 +47,7 @@ type eureka struct {
 func ReadConfig(loc string) (conf Config, err error) {
 	err = gcfg.ReadFileInto(&conf, loc)
 	if err != nil {
-		log.Criticalf("Unable to read config file Error: %s", err.Error())
+		log.Warningf("Unable to read config file Error: %s", err.Error())
```

Comment:
Can you please explain why you did this change? Unless there is a good reason I would like that to remain same as it was.

Reply:
At that time, I also used `op/go-logging` in the project(version tag v1 b2cb9fa). This version no such method `log.Criticalf`, I'm wrong to think this is the latest version, so deprecated `log.Criticalf`.

I just checked the ` op/go - logging `, found that I use is an old version, it's my fault.